File: rasterizer-dotnet-vb/MonochromeImage.vb
Imports ceTe.DynamicPDF.Rasterizer

Public Class MonochromeImage
        Friend Shared Sub Run()
            Dim rasterizer As New PdfRasterizer(Program.GetResourcePath("DocumentA.pdf"))
            Dim bmpMonochromeColorFormat As New BmpMonochromeColorFormat(DitheringAlgorithm.FloydSteinberg, 100)
            Dim bmpImageFormat As New BmpImageFormat(bmpMonochromeColorFormat)
            Dim fixedImageSize As New FixedImageSize(595, 841)
            rasterizer.Draw(Program.GetOutputDocPath("monochrom-image-output.bmp"), bmpImageFormat, fixedImageSize)
        End Sub
    End Class

File: rasterizer-dotnet-examples-csharp/MonochromeImage.cs
using ceTe.DynamicPDF.Rasterizer;

namespace rasterizer_dotnet_csharp
{
    public class MonochromeImage
    {
        internal static void Run()
        {
            PdfRasterizer rasterizer = new PdfRasterizer(Program.GetResourcePath("DocumentA.pdf"));
            BmpMonochromeColorFormat bmpMonochromeColorFormat = new BmpMonochromeColorFormat(DitheringAlgorithm.FloydSteinberg, 100);
            BmpImageFormat bmpImageFormat = new BmpImageFormat(bmpMonochromeColorFormat);
            FixedImageSize fixedImageSize = new FixedImageSize(595, 841);
            rasterizer.Draw(Program.GetOutputDocPath("monochrom-image-output.bmp"), bmpImageFormat, fixedImageSize);
        }
    }
}